apps/08jquery-towers-of-hanoi/script.js

```diff
@@ -1,5 +1,48 @@
 'use strict';
+var $block = null;
 
+// This function will execute the following code when everything in the html document has been loaded
 $(document).ready(function() {
-  // Put app logic here
+  var $stack = $('[data-stack]');
+  var $stackTwo = $('[data-stack="2"]');
+  var $stackThree = $('[data-stack="3"]');
```

Comment:
`$stackTwo` & `$stackThree` need to be globally defined if they're going to be accessed by any of your functions below.